test/connect-with-can-define.js

```diff
@@ -12,7 +12,7 @@ QUnit.module('@connect with can-define', () => {
 
 		const ViewModel = DefineMap.extend('ViewModel', {
 			init(props) {
-				assert.deepEqual(props, { bar: 'bar' }, 'constructor is called with the correct props');
+				assert.deepEqual(props, undefined, 'constructor is called with the correct props');
```

Comment:
Should we take this assertion away or at least change the description to describe why we initialize the ViewModel before assigning props?